### PR TITLE
Adding sseclient-py recipe

### PR DESCRIPTION
**Pull request by sanurielf**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sanurielf** [comment]
Hey @mpetazzoni this PR is to create a recipe in conda-forge for SSE Client. Please comment on this PR if you approve of being a maintainer (along with me and whoever joins in the future) of this.

File: recipes/sseclient-py/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - sseclient
```
Comment (mpetazzoni) [inline recipes/sseclient-py/meta.yaml:20]: @sanurielf Are you sure this should be here? `sseclient` is a _different_ Python library for SSE: https://github.com/btubbs/sseclient, and `sseclient-py` does not depend on it.

**Review by mpetazzoni — COMMENTED** [review]
One comment on the recipe Meta YAML file, but otherwise I approve of being a maintainer for this recipe alongside @sanurielf.

**Comment by sanurielf** [comment]
@conda-forge/help-python, ready for review

File: recipes/sseclient-py/meta.yaml (hunk 1 above)
Comment (sanurielf) [reply recipes/sseclient-py/meta.yaml]: Fixed, thanks!

**Pull request merged**